Makefile

```diff
@@ -8,6 +8,16 @@ BUILT_CUSTOM_RESOURCES=${PWD}/templates/custom-resources
 GOBIN=${PWD}/bin/tools
 COVERAGE=coverage.out
 
+DESTINATION=./bin/local/${BINARY_NAME}
+VERSION=$(shell git describe --always --tags)
+
+BINARY_S3_BUCKET_PATH=https://amazon-ecs-cli-v2.s3.amazonaws.com
```

Comment:
Is there an advantage of having this as a build time variable, instead of making it a constant in the code?

Reply:
We're considering putting "nightly builds" at a different path, so the build process can determine that path and inject the proper value instead of maintaining any of that logic in the codebase.